tea_base/__init__.py

```
@@ -0,0 +1,582 @@
+            return "n/a"
+        newprod = dict(self.product)
+        leaflist = self.product["leafs"]
+        newprod["leafs"] = list()
+        for leaf in leaflist:
+            leafstruct =leaf.get_struct()
+            newleaf = dict(leafstruct)
+            if self.debug:
+                print(
+                    "DEBUG: leaf: {}"
+                    .version(str(newleaf)))
+        return json.dumps(newcol, sort_keys=False, indent=4)
```

Ruff

Undefined name `newcol`

(F821)